sdk/maps/maps-route-rest/README.md

```diff
@@ -52,24 +52,24 @@ Set the values of the client ID, tenant ID, and client secret of the Microsoft E
 You will also need to specify the Azure Maps resource you intend to use by specifying the `clientId` in the client options.
 The Azure Maps resource client id can be found in the Authentication sections in the Azure Maps resource. Please refer to the [documentation](https://learn.microsoft.com/azure/azure-maps/how-to-manage-authentication#view-authentication-details) on how to find it.
 
-```javascript
-const MapsRoute = require("@azure-rest/maps-route").default;
-const { DefaultAzureCredential } = require("@azure/identity");
+```ts snippet:ReadmeSampleCreateClient_TokenCredential
+import { DefaultAzureCredential } from "@azure/identity";
+import MapRoute from "@azure-rest/maps-route";
```

Comment:
use the old name which matches package name?
```suggestion
import MapsRoute from "@azure-rest/maps-route";
```

Reply:
Fixed and redid the snippets.